tests/output/cpp/34191-comment-align-multiline.cpp

```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+
+void function()
+{
+    printf( "Hello World\n" );
+    /*
+       //test
+       /// Another comment
+       //end test
+     */
+}
```

Comment:
Without the unnecessary changes (made by the config file) the output should be this:
```cpp
#include <stdio.h>

void function()
{
	printf( "Hello World\n" );
	/*
	   //test
	   /// Another comment
	   //end test
	 */
}

```
(Note that uncrustify defaults to tab indents)